net-misc/networkmanager-fortisslvpn/networkmanager-fortisslvpn-1.2.2.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+GNOME_ORG_MODULE="NetworkManager-${PN##*-}"
+
+inherit gnome2 autotools
+
+DESCRIPTION="NetworkManager Fortinet SSLVPN compatible plugin"
+HOMEPAGE="https://wiki.gnome.org/Projects/NetworkManager"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="gtk"
+
+RDEPEND="
+	>=net-misc/networkmanager-1.2.2:=
+	>=dev-libs/dbus-glib-0.74
+	>=dev-libs/glib-2.32:2
+	>=net-misc/openfortivpn-1.2.0
+	gtk? (
+		app-crypt/libsecret
+		>=gnome-extra/nm-applet-1.0.5
+		>=x11-libs/gtk+-3.4:3
+	)
+"
+DEPEND="${RDEPEND}
+	sys-devel/gettext
+	dev-util/intltool
+	virtual/pkgconfig
+"
+
+PATCHES=(
+	"${FILESDIR}/${P}-location-fix.diff"
+)
+
+src_prepare() {
+	gnome2_src_prepare
+
+	AT_NOELIBTOOLIZE=yes eautoreconf
```

Comment:
Well, I guess this is a design defect from eapply as all eclasses are supposed to have it in their src_prepare but when inheriting multiples eclasses, you obviously risk ending up calling multiple src_prepare with eapply. I think this should be taken to a PMS bug though I won't delay merging this because of it. The simple workaround is to call eapply from src_prepare.